modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -100,11 +100,11 @@
 )
 public class CalculatePerformanceStreamProcessorExtension extends StreamProcessor {
     private static final Logger log = Logger.getLogger(CalculatePerformanceStreamProcessorExtension.class);
-    private static final int RECORDWINDOW = 5000000;
+    private static final int RECORDWINDOW = 5;
```

Comment:
**'RECORDWINDOW'**  variable indicates after how many events received, a row should be recorded in the output file. To reduce the error rate we usually maintain this value according to the event sending rate from TCP Client. (approximately 10 times bigger than the sending rate).

 In the previous implementation the value of **'RECORDWINDOW'** is higher due to the long run purpose. That's why we do not get the performance values in the output file while we are doing the Short time running experiments.